theme.ts

```diff
@@ -2,8 +2,12 @@ export const theme = {
   colors: {
     black: "#000000",
     white: "#FFFFFF",
+    white_300: "#FFFFFF33",
     black_400: "#00000040",
+    black_300: "#00000033",
+
```

Comment:
would be better to order a color next to it's shades
```suggestion
    black: "#000000",
    black_400: "#00000040",
    black_300: "#00000033",
    white: "#FFFFFF",
    white_300: "#FFFFFF33",
   

```